Tests/SwiftDocCTests/Infrastructure/DocumentationContext/DocumentationContext+RootPageTests.swift

```diff
@@ -169,4 +169,134 @@ class DocumentationContext_RootPageTests: XCTestCase {
         
         XCTAssertEqual(context.problems.count, 0)
     }
+    
+    func testWarnsAboutMultipleModuleRoots() throws {
+        let moduleOneSymbolGraph = SymbolGraph(
+            metadata: SymbolGraph.Metadata(
+                formatVersion: SymbolGraph.FormatVersion(major: 1, minor: 0, patch: 0),
+                generator: "unit-test"
+            ),
+            module: SymbolGraph.Module(
+                name: "ModuleOne",
+                platform: .init(architecture: nil, vendor: nil, operatingSystem: nil, environment: nil),
+                version: nil
+            ),
+            symbols: [],
+            relationships: []
+        )
+        
+        let moduleTwoSymbolGraph = SymbolGraph(
+            metadata: SymbolGraph.Metadata(
+                formatVersion: SymbolGraph.FormatVersion(major: 1, minor: 0, patch: 0),
+                generator: "unit-test"
+            ),
+            module: SymbolGraph.Module(
+                name: "ModuleTwo",
+                platform: .init(architecture: nil, vendor: nil, operatingSystem: nil, environment: nil),
+                version: nil
+            ),
+            symbols: [],
+            relationships: []
+        )
+        
+        let moduleOneSymbolGraphURL = try createTempFile(name: "module-one.symbols.json", content: try JSONEncoder().encode(moduleOneSymbolGraph))
+        let moduleTwoSymbolGraphURL = try createTempFile(name: "module-two.symbols.json", content: try JSONEncoder().encode(moduleTwoSymbolGraph))
+        
+        let (_, context) = try loadBundle(copying: "TestBundle", excludingPaths: ["TestBundle.symbols.json"], additionalSymbolGraphs: [moduleOneSymbolGraphURL, moduleTwoSymbolGraphURL])
```

Comment:
This code doesn't compile. Even if did, there is no "TestBundle" test bundle, so it would fail when the test ran.